lib/gtl_formatter.rb

```diff
@@ -177,9 +177,19 @@ def self.os_info_details(item)
     "svg/os-#{item.os_image_name}.svg"
   end
 
+  def self.item_provider(item)
+    type = item.class.to_s.split('::').last
```

Comment:
@Fryguy has the right solution below for this case, but just FYI in the future if you did want the base-class for some other reason you're better off with `item.class.base_class` rather than `item.class.to_s.split("::").last` because you don't have to keep the base class name the same.

For example https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/embedded_ansible/automation_manager/credential.rb#L1 `item.class.base_class` would give you `Authentication` but your `item.class.to_s.split("::").last` would give you `Credential`.